lib/grammers-mtsender/src/lib.rs

```diff
@@ -467,9 +469,8 @@ impl<T: Transport, M: Mtp> Sender<T, M> {
                             }
                         };
 
-                        drop(req);
                         let req = self.requests.remove(i);
-                        drop(req.result.send(result));
+                        req.result.send(result).unwrap();
```

Comment:
I don't think we can `unwrap` this one. The user may have dropped the oneshot receiver by the time a response from Telegram arrives. In this case we actually want to ignore. If clippy is complaining, perhaps we could `debug!()` `if .is_err()`.

Reply:
so leave it to another PR and revert or should I add the debug log?

Reply:
Please remove this change from this one and then we can send a separate one with a better "fix".

Reply:
done